client/devpi/use.py

```diff
@@ -343,6 +343,18 @@ def getvenv():
         return None
     return pip.dirpath().dirpath()
 
+def current_venv():
```

Comment:
I did switch the order around in a commit since this, I do think it's much better now.